### PR TITLE
HW_7: add router

### PR DESCRIPTION
**Pull request by Natala-Kozbina**

### CODE REVIEW

File: Angular/src/app/add-new-course/add-new-course.component.html [hunk 1]
```diff
@@ -83,7 +83,7 @@
                 </button>
 
               <button
-                  (click)="canselAddCourse()"
+                  (click)="canselAddCourse(addCourseForm.value)"
```
Comment (Nikami) [inline Angular/src/app/add-new-course/add-new-course.component.html]: Typo => cancel

Comment (Natala-Kozbina) [reply Angular/src/app/add-new-course/add-new-course.component.html]: done

File: Angular/src/app/add-new-course/add-new-course.component.ts [hunk 2]
```diff
@@ -1,4 +1,6 @@
-import {Component, EventEmitter, OnInit, Output} from '@angular/core';
+import {ChangeDetectionStrategy, Component, EventEmitter, OnInit, Output} from '@angular/core';
```
[the remark this answers is not in the document]
Comment (Natala-Kozbina) [reply Angular/src/app/add-new-course/add-new-course.component.ts]: done

File: Angular/src/app/app.component.html [hunk 3]
```diff
@@ -1,33 +1,13 @@
...
     <ng-container>
-        <app-add-new-course
-            *ngIf="isAddNewCourse"
-            (onCancelAddCourse)="cancelAddCourse()"
-            (newCourseData)="createNewCourse($event)">
-        </app-add-new-course>
+        <app-breadcrumbs></app-breadcrumbs>
     </ng-container>
 
+   <ng-container>
```
[the remark this answers is not in the document]
Comment (Natala-Kozbina) [reply Angular/src/app/app.component.html]: done

File: Angular/src/app/edit-page/edit-page.component.html [hunk 4]
```diff
@@ -1 +1,81 @@
-<p>edit-page works!</p>
+<div class="edit-page-wrapper">
+    <div class="edit-page_header">
+        Edit course
+    </div>
+
+  <form action=""
```
[the remark this answers is not in the document]
Comment (Natala-Kozbina) [reply Angular/src/app/edit-page/edit-page.component.html]: done

File: Angular/src/app/edit-page/edit-page.component.scss [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+        background-color: #f2f3f7;
+        min-height: 80vh;
+    }
+    &_header {
+        color: #03237d;
+        font-size: 26px;
+        text-align: center;
+        font-weight: 500;
+    }
+    &_title-wrapper {
```
[the remark this answers is not in the document]
Comment (Natala-Kozbina) [reply Angular/src/app/edit-page/edit-page.component.scss]: you are right - I have just removed it

File: Angular/src/app/edit-page/edit-page.component.ts [hunk 6]
```diff
@@ -1,15 +1,83 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, ChangeDetectionStrategy, ChangeDetectorRef, } from '@angular/core';
+import {ActivatedRoute, Router} from '@angular/router';
```
[the remark this answers is not in the document]
Comment (Natala-Kozbina) [reply Angular/src/app/edit-page/edit-page.component.ts]: done

File: Angular/src/app/not-found-page/not-found-page.component.ts [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+  selector: 'app-not-found-page',
+  templateUrl: './not-found-page.component.html',
+  styleUrls: ['./not-found-page.component.scss']
+})
+export class NotFoundPageComponent implements OnInit {
+
+  constructor() { }
+
+  ngOnInit() {
+  }
```
[the remark this answers is not in the document]
Comment (Natala-Kozbina) [reply Angular/src/app/not-found-page/not-found-page.component.ts]: done

File: Angular/src/app/main-page/main-list/main-course/main-course.component.ts [hunk 8]
```diff
@@ -1,10 +1,11 @@
-import { Component, OnInit, Input, Output, EventEmitter } from '@angular/core';
+import {Component, OnInit, Input, Output, EventEmitter, ChangeDetectionStrategy} from '@angular/core';
```
[the remark this answers is not in the document]
Comment (Natala-Kozbina) [reply Angular/src/app/main-page/main-list/main-course/main-course.component.ts]: done

File: Angular/src/app/edit-page/edit-page.component.ts [hunk 9]
```diff
@@ -1,15 +1,83 @@
...
+              updateOn: 'blur'
+          }],
+          time: [this.currentCourse.time, {
+              validators: [Validators.required],
+              updateOn: 'blur'
+          }],
+      });
+  }
+
+  canselEditCourse() {
```
[the remark this answers is not in the document]
Comment (Natala-Kozbina) [reply Angular/src/app/edit-page/edit-page.component.ts]: done

**Pull request merged**